tasks/main.yml

```diff
@@ -98,6 +98,17 @@
   when:
     - deploydrupal_site_install
 
+- name: adjust files directory sub-folder permissions after site install.
+  file:
+    path: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}/files"
+    state: directory
+    owner: "{{ deploydrupal_apache_user }}"
+    group: "{{ deploydrupal_checkout_user }}"
+    mode: 0770
+    recurse: yes
+  become: "{{ deploydrupal_files_become }}"
+  when: deploydrupal_site_install_config
```

Comment:
I guess this only occurs when installing with config since that is how the media module will be installed? What if someone is installing with a profile other than minimal?

Reply:
I wonder if we can remove the other [similar instance of this](https://github.com/ChromaticHQ/ansible-deploy-drupal/blob/file-directory-permissions/tasks/main.yml#L72-L79) and then ditch the `when` on this task so that it runs every time.

Reply:
> I guess this only occurs when installing with config since that is how the media module will be installed? What if someone is installing with a profile other than minimal?

I'm not sure where you are going with this. Let's discuss on the coffee zoom.

> I wonder if we can remove the other similar instance of this and then ditch the when on this task so that it runs every time.

Let's discuss this on coffee zoom too.

Reply:
Discussed and decided to use an array of subdirectory names to avoid altering other files.